nautobot/dcim/forms.py

```diff
@@ -2646,10 +2697,6 @@ def __init__(self, *args, **kwargs):
 
         # Limit LAG choices to interfaces belonging to this device or a peer VC member
         device = Device.objects.get(pk=self.initial.get("device") or self.data.get("device"))
-        device_query = Q(device=device)
-        if device.virtual_chassis:
-            device_query |= Q(device__virtual_chassis=device.virtual_chassis)
-        self.fields["lag"].queryset = Interface.objects.filter(device_query, type=InterfaceTypeChoices.TYPE_LAG)
```

Comment:
This also seems like a loss of functionality - can no longer assign to a LAG belonging to a different device in the same virtual-chassis?

Reply:
IMO this is not right because if a user decides to change the selected device.
It still shows lag choices meant for the previous device